src/main/java/pages/FormsPage.java

```diff
@@ -12,9 +17,90 @@ public FormsPage() {
     @AndroidFindBy(xpath = "//android.widget.ScrollView[@content-desc='Forms-screen']/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup[1]/android.widget.TextView")
     private MobileElement titleForm;
 
-    public String getTitleFormsMenu(){
+    @AndroidFindBy(accessibility = "text-input")
+    private MobileElement inputField;
+
+    @AndroidFindBy(accessibility = "input-text-result")
+    private MobileElement inputTextResult;
+
+    @AndroidFindBy(accessibility = "switch")
+    private MobileElement switchFormn;
+
+    @AndroidFindBy(accessibility = "switch-text")
+    private MobileElement switchTextFormn;
+
+    @AndroidFindBy(accessibility = "select-Dropdown")
+    private MobileElement dropDownFormn;
+
+    @AndroidFindBy(id = "android:id/text1")
+    private List<MobileElement> listOptionsDownFormn;
+
+    @AndroidFindBy(accessibility = "button-Inactive")
+    private MobileElement inactiveButton;
+
+    @AndroidFindBy(accessibility = "button-Active")
+    private MobileElement activeButton;
+
+    @AndroidFindBy(id = "android:id/button")
+    private List<MobileElement> listButtonsAlert;
+
+    private String classButton = "new UiSelector().className(\"android.widget.TextView\")";
+
+    public String getTitleFormsMenu() {
         wait.untilElementIsVisible(timeOutSeconds, titleForm);
         return titleForm.getText();
     }
 
+    public void sendTextInputField(String text) {
+        inputField.sendKeys(text);
+    }
+
+    public String getTextInputResult() {
+        return inputTextResult.getText();
+    }
+
+    public String statusSwitch() {
+        return switchFormn.getText();
+    }
+
+    public String getSwitchText() {
+        return switchTextFormn.getText();
+    }
+
+    public void clickSwitchForm() {
+        switchFormn.click();
+    }
+
+    public void selectOptionDropDown(String text) {
+        dropDownFormn.click();
+        Optional<MobileElement> option = listOptionsDownFormn.stream().filter(element -> element.getText().equals(text)).findFirst();
```

Comment:
Is it necessary to use `stream` method?

Reply:
A foreach with a flag can also be used. But they do practically the same